src/frontend/src/handler/create_table.rs

```diff
@@ -1054,6 +1078,7 @@ pub async fn generate_stream_graph_for_table(
                 source_watermarks,
                 col_id_gen,
                 append_only,
+                vec![],
```

Comment:
For some reason the real parameter is not passed but a dummy `vec![]` is provided here, leading to bug #19384.